configs/schemas/config.schema.json

```diff
@@ -312,6 +312,11 @@
           "default": false,
           "title": "Code Search Tool",
           "type": "boolean"
+        },
+        "use_dependency_checker": {
+          "default": true,
+          "title": "Whether to use VulnerableDependencyChecker tool in the pipeline or not",
```

Comment:
Minor request to update the schema file to the following auto-generated version:

```suggestion
          "description": "Whether to use the VulnerableDependencyChecker in the pipeline or not.",
          "title": "Use Dependency Checker",
```

Sorry, I realized the command I shared earlier had the incorrect `--output_file` path. I just made this [fix](https://github.com/NVIDIA-AI-Blueprints/vulnerability-analysis/commit/8eb19f57d03d717210fe9e68cdd64533b6259c24) so that the default command without any output file will save to the correct location going forward.

```
python src/main.py cve tools gen-config-schema
```